app/views/candidate_mailer/declined_by_default_without_rejections.html.erb

```diff
@@ -0,0 +1,25 @@
+Dear <%= @application_form.first_name %>,
+
+# <%= 'Application'.pluralize(@declined_course_names.size) %> withdrawn automatically
+
+<% if FeatureFlag.active?('covid_19') %>
+You did not respond in time so we declined your <%= 'offer'.pluralize(@declined_course_names.size) %> for <%= @declined_course_names.to_sentence %>.
+<% else %>
+You did not respond within <%= @declined_courses.first.decline_by_default_days %> working days so we declined your <%= 'offer'.pluralize(@declined_course_names.size) %> for <%= @declined_course_names.to_sentence %>
+<% end %>
+
+#Are you still interested in teaching?
+
+You didn’t pursue your teacher training application, but it’s not too late to apply again for a course if you’ve changed your mind.
+
+We’ve saved your last application, so all you have to do is choose a course, review your application and re-submit.
+
+Sign in to apply again:
+https://www.apply-for-teacher-training.education.gov.uk/candidate/account
+
+# Get help
+
+For advice on becoming a teacher, call Get Into Teaching for free on 0800 389 2500 or chat to an adviser online:
+https://getintoteaching.education.gov.uk/contact
+
+For any technical issues using Apply for teacher training, contact us at [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk)
```

Comment:
Some of this content can be turned into partials and reused across multiple emails. Could we try that?

Reply:
Sure i'm going to do it in the next PR and merge them together